neuralprophet/logger.py

```diff
@@ -0,0 +1,54 @@
+# https://pytorch-lightning.readthedocs.io/en/stable/extensions/logging.html#make-a-custom-logger
+import collections
+
+from pytorch_lightning.loggers.logger import Logger
+from pytorch_lightning.utilities.distributed import rank_zero_only
+from pytorch_lightning.loggers.logger import rank_zero_experiment
+
+
+class MetricsLogger(Logger):
+    def __init__(self):
+        super().__init__()
+
+        self.history = collections.defaultdict(list)
+        self.checkpoint_path = None
```

Comment:
Info: The default Lightning loggers do not maintain the provided logs in-memory but directly store them to a file. Since we want to access the metrics directly after training and not necessarily want to persist them, I defined a custom logger that maintains the logs in-memory. This allows us to access them right after training by calling `logger.history`